docs/utils/script.py

```diff
@@ -130,10 +130,9 @@ def create_file_index() -> dict:
                 continue
 
             if file.find(module) == 0:
-                new_name = remove_module_prefix_from_file(file, module)
```

Comment:
This is the current setup, where we remove_module_prefix and rename file, which causes the error with overview file